### PR TITLE
Update to 27, Gradle 4.6, remove buildToolsVersion.

### PR DESCRIPTION
**Pull request by NightlyNexus**
can break into separate commits if necessary.

### CODE REVIEW

**Comment by JakeWharton** [comment]
Need a rebase

**Comment by JakeWharton** [comment]
Looks like license hash needs updated too

**Comment by NightlyNexus** [comment]
ah, yeah, will look in a bit!

File: .travis.yml [hunk 1]
```diff
@@ -4,15 +4,19 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-26.0.2
-    - android-26
+    - android-27
```
Comment (JakeWharton) [inline .travis.yml]: You can remove this. It will be installed automatically.

Comment (NightlyNexus) [reply .travis.yml]: varynice.

**Pull request merged**